### PR TITLE
Feature/resolving event

### PR DESCRIPTION
**Pull request by NombusoRR**
Merge ResolvingEvent feature to dev

### CODE REVIEW

File: WaterLog_Backend/ControllerService.cs [hunk 1]
```diff
@@ -15,6 +15,7 @@ public class ControllerService : IControllerService
         private SegmentEventsController segmenteventsController;
         private SegmentLeaksController segmentleaksController;
         private SegmentsController segmentsController;
+        private HistoryLogController historylogController;
```
Comment (CeeyarRbt) [inline WaterLog_Backend/ControllerService.cs]: not a train smash but please check naming conversion 

~segmentleaksController~ -> `segmentLeaksController`

File: WaterLog_Backend/Controllers/HistoryLogController.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+        private readonly DatabaseContext _db;
+        readonly IConfiguration _config;
+        public HistoryLogController(DatabaseContext context, IConfiguration config)
+        {
+            _db = context;
+            _config = config;
+        }
+
+        // GET api/values
```
Comment (CeeyarRbt) [inline WaterLog_Backend/Controllers/HistoryLogController.cs]: what values are we getting?

Comment (CeeyarRbt) [reply WaterLog_Backend/Controllers/HistoryLogController.cs]: please check other places for this too.

**Review by CeeyarRbt — CHANGES_REQUESTED** [review]

File: WaterLog_Backend/Controllers/HistoryLogController.cs [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<HistoryLogEntry>>> Get()
```
Comment (FredJEhlers) [inline WaterLog_Backend/Controllers/HistoryLogController.cs:27]: What are you getting? 
Please more descriptive function names

Comment (FredJEhlers) [reply WaterLog_Backend/Controllers/HistoryLogController.cs:27]: empty line at EOF

Comment (FredJEhlers) [reply WaterLog_Backend/Controllers/HistoryLogController.cs:27]: Re-look at all api end point function names

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: WaterLog_Backend/Controllers/HistoryLogController.cs (hunk 3 above)
Comment (FredJEhlers) [reply WaterLog_Backend/Controllers/HistoryLogController.cs:27]: Nevermind @CeeyarRbt got them all 
I will resolve my comment

File: WaterLog_Backend/ControllerService.cs (hunk 1 above)
Comment (NombusoRR) [reply WaterLog_Backend/ControllerService.cs]: Thank you, fixed.

**Review by CeeyarRbt — APPROVED** [review]

File: WaterLog_Backend/Controllers/HistoryLogController.cs (hunk 2 above)
Comment (NombusoRR) [reply WaterLog_Backend/Controllers/HistoryLogController.cs]: Fixed

**Review by FredJEhlers — APPROVED** [review]

**Pull request merged**